2022/0xxx/CVE-2022-0742.json

```diff
@@ -1,18 +1,89 @@
 {
```

Comment:
'https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=2d3916f3189172d5c69d33065c3c21119fe539fc ' does not match '^(ftp|http)s?://\\S+$'

Reply:
Done